ehitchhiking-ui/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please add .vscode folder to gitignore. Git should forget about this file. Feel free to ask Gleb how to do that.

Reply:
done